phing/build.yml

```diff
@@ -67,9 +67,9 @@ drupal:
 
 drush:
   bin: ${composer.bin}/drush
-  cmd: ${drush.bin} @${drush.alias} -l ${multisite.name}
+  cmd: ${drush.bin} @${drush.alias} -l default
   dir: ${docroot}
-  uri: ${multisite.name}
+  uri: default
```

Comment:
This change should have no effect for single-site projects. Multisite projects should pass either a unique cloud alias or override `drush.uri`, for instance via example.multisite.yml.